three.html

```diff
@@ -117,21 +117,30 @@ <h2>Daily Challenge</h2>
       </aside>
       <main class="col-9">
         <ul class="points">
-          <li>Create a new repository named "bootstrap-boilerplate"</li>
-          <li>Clone your repository</li>
-          <li>Create a new branch called "starter-files"</li>
+          <li>Daily Challenge Instructions</li>
```

Comment:
I think `Daily Challenge Instructions` should have a colon at the end. The same with `Github Instructions` below this. That would be:
Daily Challenge Instructions -> Daily Challenge Instructions:
Github Instructions -> Github Instructions:

Reply:
This happens here and on Day 5, 6 and 7

Reply:
<img width="394" alt="Screen Shot 2022-10-04 at 11 13 57 AM" src="https://user-images.githubusercontent.com/10934485/193895119-9cdb1b9d-abc5-4bd9-9127-859926b222b6.png">